apis/swagger.yml

```diff
@@ -3165,25 +3176,41 @@ definitions:
         format: "int64"
         x-nullable: false
 
-  SearchResultItem:
+  SearchResults:
```

Comment:
why we need this？

Reply:
for example, we request `https://registry.hub.docker.com/v1/search?q=nginx`
it return 
```
{
	"num_pages": 2299,
	"num_results": 57454,
	"page_size": 25,
	"page": 1,
	"query": "nginx",
	"results": [{
		"name": "nginx",
		"description": "Official build of Nginx.",
		"star_count": 11436,
		"is_trusted": false,
		"is_automated": false,
		"is_official": true
	},]
}
```
`SearchResults` is used to transfer search result
```
type SearchResults struct {
	NumResults int64 `json:"num_results,omitempty"`
	Query string `json:"query,omitempty"`
	Results []*SearchResultItem `json:"results"`
}
```